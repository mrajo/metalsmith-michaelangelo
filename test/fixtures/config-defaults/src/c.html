---
title: wut
---
<h3>Nothing happens</h3>