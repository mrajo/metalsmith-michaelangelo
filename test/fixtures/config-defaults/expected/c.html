<h3>Nothing happens</h3>